errors/arsenalErrors.json

```
@@ -26,7 +26,7 @@
   },
   "BucketAlreadyOwnedByYou": {
     "code": 409,
-    "description": "Your previous request to create the named bucket succeeded and you already own it. You get this error in all AWS regions except US Standard, us-east-1. In us-east-1 region, you will get 200 OK, but it is no-op (if bucket exists S3 will not do anything)."
+    "description": "A bucket with this name already exists and owned by you"
```

What do you think of:

```suggestion
    "description": "A bucket with this name exists and is already owned by you"
```

To be closer to the name of the error?

---

fixed